app/grandchallenge/core/templatetags/meta_attr.py

```diff
@@ -7,3 +7,18 @@
 def meta_attr(obj, key):
     """Returns an attribute of an objects _meta class"""
     return getattr(obj._meta, key, "")
+
+
+@register.filter
+def verbose_name(obj):
+    return obj._meta.verbose_name
+
+
+@register.filter
+def verbose_name_plural(obj):
+    return obj._meta.verbose_name_plural
+
+
+@register.filter
+def model_name(obj):
+    return obj._meta.model_name
```

Comment:
This is the same as in the other open PR. Depending on which PR gets merged first, I will update the remaining PR.